docs/lookout-tool.md

```diff
@@ -1,22 +1,22 @@
-# lookout-sdk Binary
+# lookout-tool Binary
 
-_For the **lookout-sdk** library to develop new analyzers go to [**lookout-sdk**](https://github.com/src-d/lookout-sdk) repository._
+_For the **lookout-tool** library to develop new analyzers go to [**lookout-tool**](https://github.com/src-d/lookout-tool) repository._
```

Comment:
this line should not change at all, it's about the lookout-sdk repository.

Reply:
thanks